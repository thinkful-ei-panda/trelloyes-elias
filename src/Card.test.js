import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('<Card />',() => {
  
  const props = {
    key: "test-id",
    title: "Card title",
    content: "Card content"
  }

  it('renders without crashing', () => {

    const div = document.createElement('div');

    ReactDOM.render(<Card {...props} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Card {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });


});