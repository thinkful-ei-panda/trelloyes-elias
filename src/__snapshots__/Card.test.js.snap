// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
     delete the internet
  </button>
  <h3>
    Card title
  </h3>
  <p>
     
    Card content
  </p>
</div>
`;
